src/Tests/dotnet-workload-install.Tests/GivenDotnetWorkloadInstall.cs

```diff
@@ -37,7 +37,8 @@ public GivenDotnetWorkloadInstall(ITestOutputHelper log) : base(log)
             _manifestPath = Path.Combine(_testAssetsManager.GetAndValidateTestProjectDirectory("SampleManifest"), "Sample.json");
         }
 
-        [Fact]
+        // These two tests hit an IOException when run in helix on non-windows
+        [WindowsOnlyFact]
```

Comment:
should we track these in an issue?

Reply:
I wanted to chat with @dsplaisted first to determine how critical these tests were on non-windows as well as if he had any alternatives for solving this. The exception was coming from https://github.com/dotnet/sdk/blob/09b31215867d1ffe4955fd5b7cd91eb552d3632c/src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallerFactory.cs#L82 because the test didn't have access to write the metadata folder. I might be able to just catch the exception though I'm not sure if it'll just hit the GracefulException later in the flow instead.